doc/Settings.md

```diff
@@ -67,6 +67,20 @@ The `scope` behavior affects the choice between installing a package for the cur
     },
 ```
 
+### Locale
+
+The `locale` behavior affects the choice of installer based on installer locale. The matching parameter is `--locale`, and uses bcp47 language tag.
+
+Note the locale preference will also be applied to `winget show` command if applicable.
```

Comment:
Should they?  It feels like a separate setting should exist for display language.